ts/store/reducers/wallet/wallets.ts

```diff
@@ -533,12 +545,24 @@ const reducer = (
     //
     // set favourite wallet
     //
-
+    case getType(setFavouriteWalletRequest):
+      return {
+        ...state,
+        updatingFavouriteWallet: pot.someLoading(action.payload)
+      };
+    case getType(setFavouriteWalletFailure):
+      return {
+        ...state,
+        updatingFavouriteWallet: pot.isSome(state.updatingFavouriteWallet)
+          ? pot.someError(state.updatingFavouriteWallet.value, action.payload)
+          : pot.noneError(action.payload)
+      };
```

Comment:
There is a specific reason why you didn't use the `toSome` method?
I think that this can simplify the code, do you agree?
```suggestion
    case getType(setFavouriteWalletFailure):
      return {
        ...state,
        updatingFavouriteWallet: pot.toError(
          state.updatingFavouriteWallet,
          action.payload
        )
      };
```

Reply:
Surely better!
Thank you
a2fc022fd4a7d5df568dc72bfedc811eaa1a906a